### PR TITLE
[REVIEW] Colab transition

### PR DESCRIPTION
**Pull request by taureandyernv**
transitioning colab support into notebooks repo

### CODE REVIEW

**Comment by taureandyernv** [comment]
@randerzander please review

**Comment by taureandyernv** [comment]
@randerzander, not tested on premium yet.  

The differences (from notebooks-contrib/branch-0.12) are:
- new migration notice, since most people are pointing to notebooks contrib
- removable of support for notebooks before 0.11 (pyarrow will become default, but that will need to be refactored some more as this had a quick turn around)
- the normal progression of what is in staging and what is in nightlies.

File: utils/rapids-colab.sh [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            python=3.6 cudatoolkit=10.0 \
+            cudf=$RAPIDS_VERSION cuml cugraph cuspatial gcsfs pynvml xgboost\
+            dask-cudf
+        # check to make sure that pyarrow is running the right version (0.15) for v0.11 or later
+        wget -nc https://github.com/rapidsai/notebooks/raw/master/utils/update_pyarrow.py
+    fi
+      
+    echo "Copying shared object files to /usr/lib"
+    # copy .so files to /usr/lib, where Colab's Python looks for libs
+    cp /usr/local/lib/libcudf.so /usr/lib/libcudf.so
```
Comment (randerzander) [inline utils/rapids-colab.sh]: I believe you need to copy .so files from cuml, cugraph, xgboost, cuspatial into /usr/lib as well.

**Comment by taureandyernv** [comment]
Closing as we decided to put this in its own repo

**Pull request closed without merging**